sdk/notificationhubs/notification-hubs/src/auth/hmacSha256.browser.ts

```diff
@@ -1,68 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-// stubs for browser self.crypto
-interface JsonWebKey {}
-
-interface CryptoKey {}
-
-type KeyUsage =
-  | "decrypt"
-  | "deriveBits"
-  | "deriveKey"
-  | "encrypt"
-  | "sign"
-  | "unwrapKey"
-  | "verify"
-  | "wrapKey";
-
-interface Algorithm {
-  name: string;
-}
-
-interface SubtleCrypto {
-  importKey(
-    format: string,
-    keyData: JsonWebKey,
-    algorithm: HmacImportParams,
-    extractable: boolean,
-    usage: KeyUsage[]
-  ): Promise<CryptoKey>;
-  sign(
-    algorithm: HmacImportParams,
-    key: CryptoKey,
-    data: ArrayBufferView | ArrayBuffer
-  ): Promise<ArrayBuffer>;
-  digest(algorithm: Algorithm, data: ArrayBufferView | ArrayBuffer): Promise<ArrayBuffer>;
-}
-
-interface Crypto {
-  readonly subtle: SubtleCrypto;
-  getRandomValues<T extends ArrayBufferView | null>(array: T): T;
-}
-
-declare const self: {
-  crypto: Crypto;
-};
-
-interface HmacImportParams {
-  name: string;
-  hash: Algorithm;
-  length?: number;
-}
-
-// stubs for browser TextEncoder
-interface TextEncoder {
-  encode(input?: string): Uint8Array;
-}
-
-// eslint-disable-next-line @typescript-eslint/no-redeclare
-declare const TextEncoder: {
-  prototype: TextEncoder;
-  new (): TextEncoder;
-};
-
-declare function btoa(str: string | Buffer): string;
+/// <reference lib="dom" />
```

Comment:
@witemple-msft didn't we have some other way of doing this where we edited the types includes or something?